sklearn/ensemble/bagging.py

```diff
@@ -52,6 +53,11 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
     bootstrap_features = ensemble.bootstrap_features
     support_sample_weight = has_fit_parameter(ensemble.base_estimator_,
                                               "sample_weight")
+    # Logistic regression does not support sample weights with liblinear
+    if (isinstance(ensemble.base_estimator_,LogisticRegression) and 
```

Comment:
Sorry, I should say test *error* not failure. See line 1509 here:
https://travis-ci.org/scikit-learn/scikit-learn/jobs/78473976

Essentially: In the past `support_sample_weight` would be `False` and `_parallel_build_estimators`  (in bagging.py) would work as required. Now however it's `True` (since it's a parameter in the `fit` routine) and is called with `liblinear` + `sample_weight`, which yields the new error I added at line 416 in logistic.py (can't use `liblinear` with `sample_weight`).